orion/pipelines/verified/tadgan/tadgan_encoder_downsample.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
I would suggest to copy and paste tadgan folder to sandbox and start any experimental changes there.